tools/importer/src/main/java/com/vesoft/nebula/tools/ClientManager.java

```diff
@@ -0,0 +1,31 @@
+package com.vesoft.nebula.tools;
+
+import com.google.common.net.HostAndPort;
+import com.vesoft.nebula.graph.client.GraphClient;
+import com.vesoft.nebula.graph.client.GraphClientImpl;
+import org.apache.log4j.Logger;
+
+import java.util.List;
+
+public class ClientManager {
+    private static final Logger LOGGER = Logger.getLogger(ClientManager.class.getClass());
+
+    private static ThreadLocal<GraphClient> clientThreadLocal = new ThreadLocal<>();
```

Comment:
So the threads will share the one GraphClient？ How to make sure thread safe？

Reply:
**ThreadLocal** makes sure that each thread in threadpool will hold unique instance.